percy/providers/app_automate.py

```diff
@@ -40,13 +40,15 @@ def _get_tiles(self, **kwargs):
             return super()._get_tiles(**kwargs)
         screen_lengths = kwargs.get('screen_lengths', 4)
         scrollable_xpath = kwargs.get('scollable_xpath')
+        force_full_page = kwargs.get('force_full_page')
```

Comment:
This should an env var and not an arg as discussed before, if we add an arg we will be forced to support it forever

Reply:
moved to env variable